contrib/IECoreMantra/src/IECoreMantra/ProceduralPrimitive.cpp

```diff
@@ -197,7 +203,7 @@ void IECoreMantra::ProceduralPrimitive::addVisibleRenderable( VisibleRenderableP
 		m_postBlur = -m_cameraShutter[1] / m_fps;
 		openGeometryObject();
 			addGeometry(gdp, 0.0f);
-			addVelocityBlurGeometry(gdp, m_preBlur, m_postBlur);
```

Comment:
The whole openGeometryObject was deprecated in 15, so these functions exist in 15. How would I test this? In FX, we're using bgeo caches, and Houdini's delayed load instead.